Development environment setup/WSL Setup.md

```diff
@@ -207,10 +251,12 @@ Make sure you replace the `YOURUSERNAME` with your actual username. From the con
 
 The above template will be used to set up a new site.
 
-Whenever you are making changes to your Nginx config, you can test it using sudo `nginx -t` command. If you make an error, the test command will point it out.
+Whenever you are making changes to your Nginx config, you can test it using `sudo nginx -t` command. If you make an error, the test command will point it out.
 
 ### MySQL
 
+⚠️ Make sure you don't have MySQL installed on the system already (from some out of the box solutions like Local), as it will conflict with the WSL's installation and you probably won't be able to get it to work correctly.
```

Comment:
```suggestion
⚠️ Make sure you don't have MySQL installed on the system already (from some out-of-the-box solutions like Local), as it will conflict with the WSL's installation and you probably won't be able to get it to work correctly.
```